packages/integrations/manager.js

```diff
@@ -57,23 +57,9 @@ class IntegrationManager extends Delegate {
 
         const instance = await integrationManagerClass.getInstance({
             userId,
-            integrationId: params.integrationId,
+            events: integrationManagerClass.Config.events
```

Comment:
Is there any chance this will impact previous implementations?

Reply:
@MichaelRyanWebber  thanks for the review. Just for reference this PR is pretty old and doesn't indicate the bigger refactor I brought up in [slack](https://lefthookhq.slack.com/archives/C017RNCQV70/p1687460445053899)

This shouldn't cause a problem for previous implementations, even though it looks like it. While getInstance was passed the integrationId, it didn't actually do anything with it. It passed down into the Delegate constructor and ignored. 

In terms of the events, while this change means getInstance now supports supplying events on instantiation, it does not require it and reverts to the previous behavior if they are absent. 

That said, this was all rather confusing to begin with, and could use more work that could stray into breaking changes. The existing Frigg Apps i've seen tend to override the `getInstanceFromIntegrationId` but continue to use the base class implementation of `getInstance`, so I held that as a limitation for now.

Reply:
I do think the change to getInstance and the Delegate constructor could be problematic. Technically the previous iteration instantiated two `IntegrationManager`s with the first as the second's Delegate. The first gets supplied with events (i.e. delegateTypes) but the second does not as part of `getInstance`.

The approach I switched to makes the `delegate` point back to itself by default. But there's a possibility I misunderstand the intentions or all the use-cases involved. Thoughts welcome!